CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,4 @@
 - Fix: Correct rendering of response to convop "POST /v1/contextEntities/{entityId::id}/attributes/{attributeName} (Issue #772)
-- Fix:  All convenience operations to use the service routines of standard operations (Issue #117)
+- Fix: All convenience operations to use the service routines of standard operations (Issue #117)
+- Fix: Made 'POST /v1/contextEntities/{entityId::id}' and 'POST /v1/contextEntities' accept entity::id and entity::type in payload,
```

Comment:
I would say in the opossite way at the "right way" in conv ops should be the URL. I mean:

"Made 'POST /v1/contextEntities/{entityId::id}' and 'POST /v1/contextEntities' enity::id and entity::type in the payload, as long as they coincide with the entityId::id and entityId::type in the URL"